discord/time.go

```diff
@@ -32,7 +32,7 @@ func (t *Time) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
-	t.Time = parsed
+	t.Time = parsed.UTC()
```

Comment:
it seems discord does not specify their timezone, because ISO8601 includes timezone information. I just added `.UTC()` to pass the test. `.UTC()` just changes timezone in `time.Time` instance, so the absolute time will not be changed.

Reply:
perhaps we should instead update the test?